certificate-service/app/src/main/resources/fk7809/schematron/lumek.v1.sch

```diff
@@ -0,0 +1,413 @@
+          Ett 'LUMEK' får ha högst ett 'Funktionsnedsättning för sinnesfunktioner och smärta
+        </iso:assert>
+       <iso:assert test="count(gn:svar[@id='13']) le 1">
+        Ett 'LUMEK' får ha högst ett 'Funktionsnedsättning för balans, koordination och motorik'
+       </iso:assert>
+       <iso:assert test="count(gn:svar[@id='14']) le 1">
+        Ett 'LUMEK' får ha högst ett 'Funktionsnedsättning för annan kroppslig funktion'
+       </iso:assert>
+       <iso:assert test="count(gn:svar[matches(@id, '^(48|49|[89]|1[0-4])$')]) ge 1">
+        Ett 'LUMEK' måste ha minst ett 'Funktionsnedsättning'
+       </iso:assert>
+    </iso:rule>
+  </iso:pattern>
+
+  <iso:pattern id="q1">
+   <iso:rule context="//gn:svar[@id='1']">
+      <iso:assert test="count(gn:instans) = 1">
+        'Grund för medicinskt underlag (MU)' måste ha ett instansnummer.
+      </iso:assert>
+      <iso:assert test="count(gn:delsvar[@id='1.1']) = 1">
+          'Grund för medicinskt underlag (MU)' måste ha ett 'Typ av grund för MU'.
+      </iso:assert>
+      <iso:assert test="count(gn:delsvar[@id='1.2']) = 1">
+         'Grund för medicinskt underlag (MU)' måste ha ett 'Datum som grund för MU'.
+      </iso:assert>
+      <iso:assert test="count(gn:delsvar[@id='1.3']) le 1">
+          'Grund för medicinskt underlag (MU)' får ha högst ett 'Ange vad annat är' svar.
+      </iso:assert>
+            <iso:assert test="count(gn:delsvar[@id='1.4']) le 1">
+                'Grund för medicinskt underlag (MU)' får ha högst ett 'Ange anhörig eller annas relation till patienten' svar.
+            </iso:assert>
+      <iso:assert test="not(preceding-sibling::gn:svar[@id='1']/gn:delsvar[@id='1.1']/tp:cv/tp:code/normalize-space() = normalize-space(gn:delsvar[@id='1.1']/tp:cv/tp:code))">
+        Samma 'Typ av grund för MU' kan inte användas flera gånger i samma 'LUMEK'.
+      </iso:assert>
+   </iso:rule>
+  </iso:pattern>
+
+  <iso:pattern id="q1.1">
+    <iso:rule context="//gn:delsvar[@id='1.1']">
+      <iso:extends rule="cv"/>
+      <iso:assert test="tp:cv/tp:codeSystem = 'KV_FKMU_0001'">'codeSystem' måste vara 'KV_FKMU_0001'.</iso:assert>
+      <iso:assert test="matches(normalize-space(tp:cv/tp:code), '^(UNDERSOKNING|JOURNALUPPGIFTER|ANNAT|ANHORIG)$')">
+        'Typ av grund för MU' kan ha ett av värdena UNDERSOKNING, JOURNALUPPGIFTER, ANNAT, ANHORIG.
+      </iso:assert>
+    </iso:rule>
+  </iso:pattern>
+
+  <iso:pattern id="q1.2">
+    <iso:rule context="//gn:delsvar[@id='1.2']">
+      <iso:extends rule="date"/>
+    </iso:rule>
+  </iso:pattern>
+
+  <iso:pattern id="q1.3">
+    <iso:rule context="//gn:delsvar[@id='1.3']">
+      <iso:extends rule="non-empty-string"/>
+      <iso:assert test="count(../gn:delsvar[@id='1.1']/tp:cv/tp:code[normalize-space(.) != 'ANNAT']) = 0">
+        Om 'Typ av grund för MU' inte är 'Annat' så får 'Ange vad annat är' inte anges.
+      </iso:assert>
+    </iso:rule>
+  </iso:pattern>
+
+  <iso:pattern id="q1.1-1.4">
```

Comment:
Should this be 1.1-1.4 or 1.1-1.3?

Reply:
1.1-1.3 makes sense since the pattern is testing delsvar 1.3. Seems like we already defined a 1.1-1.4 on line 134 to test the 1.4 delsvar!

Reply:
Okay but isn't it weird then if the test for 1.4 includes 1.3?